web-animations-2/Overview.bs

```diff
@@ -3366,6 +3553,167 @@ Update the type for the AnimationPlaybackEventInit members.
 
 </div>
 
+<h3 id="the-animationtrigger-interface">The <code>AnimationTrigger</code> interface</h3>
```

Comment:
I think you still need to add AnimationTrigger to the animation interface: https://drafts.csswg.org/web-animations-2/#the-animation-interface as well as the KeyframeAnimationOptions used when constructing an animation: https://drafts.csswg.org/web-animations-1/#dictdef-keyframeanimationoptions

This would allow for an animation to be setup via the web-animations api doing something like this:
```js
element.animate({
    // slide and fade in
    transform: ['translateY(-50px)', 'none'],
    opacity: [0, 1]}, {
    duration: 1000,
    // Starts when the trigger point is reached.
    trigger: new AnimationTrigger(...)};
```

Note that trigger can be null, and if null you could apply a default trigger similar to how we use the default document timeline if timeline is null: https://drafts.csswg.org/web-animations-1/#dictdef-keyframeanimationoptions:~:text=if%20timeline%20member%20of%20options%20is%20missing

Reply:
In case it's null I think I rather keep it that way and not add a default trigger. Just keep the existing behavior. that should keep consistency with creating an Animation via constructor and inspecting the object.
WDYT?